aioamqp/__init__.py

```diff
@@ -39,12 +39,11 @@
 
     create_connection_kwargs = {}
 
-    if ssl:
-        ssl_context = ssl_module.create_default_context()
+    if ssl is not None:
         if not verify_ssl:
-            ssl_context.check_hostname = False
-            ssl_context.verify_mode = ssl_module.CERT_NONE
-        create_connection_kwargs['ssl'] = ssl_context
+            ssl.check_hostname = False
```

Comment:
We may want to remove the verify_ssl parameter, since the context can be configured in the parent code

Reply:
I agree - have removed :)